src/Integration/Binding/SolutionBindingOperation.cs

```diff
@@ -256,6 +257,20 @@ private void AddFileToSolutionItems(string fullFilePath)
             }
         }
 
+        private void RemoveFileFromSolutionItems(string fullFilePath)
+        {
+            Debug.Assert(Path.IsPathRooted(fullFilePath) && this.sourceControlledFileSystem.FileExist(fullFilePath), "Expecting a rooted path to existing file");
+
+            Project solutionItemsProject = this.projectSystem.GetSolutionItemsProject(false);
```

Comment:
Looking at this line it's not clear what `false` is. Adding `createOnNull: false` would help readability a lot.